Add docs about diagnosing and debugging hardware and video acceleration (#180)
foxglove/docs: docs/3-visualization/4-panels/image.md

 - Disable frame reordering on your encoder.
 - Lower profile levels (`level_idc`) usually require smaller buffers, resulting in lower-latency decoding.
 - If your encoder allows it, use a `bitstream_restriction` in the VUI parameters to limit the size of the buffer (`max_dec_frame_buffering`) and also disable frame reordering (`max_num_reorder_frames`).

If you see that the frame delay increases after pausing that likely means that the video decoder can't keep up with your stream, and you'll want to reduce the number of your panels displaying video streams and/or check that your platform supports video acceleration. Read on to see more information.

System CPU load and power consumption may also contribute to decoding delays. Hardware-accelerated decoding is generally faster and more energy-efficient. Make sure to check that it is enabled.

These steps can help your platform achieve low-latency decoding, but there is no guarantee depending on the platform and device being used.

> ## Checking video acceleration:

Reviewer: ```suggestion
## Checking hardware acceleration
```

Author: ah yes will fix this to say "Checking webGL hardware acceleration".

Author: same response as below, but good suggestion

> ## Checking video acceleration:

Reviewer: ```suggestion
## Checking video acceleration
```

>  You can check that your platform supports hardware-accelerated decoding by opening Google Chrome and entering `chrome://gpu` in the address bar. If you do not see `Video Decode: Hardware accelerated` or the `Video Acceleration` section is empty on this page, but you believe that it should be supported, then you may need to take additional platform-specific steps to enable it.
> 
>  There's no way to check this in our desktop app yet. However, because our desktop app uses Electron (a Chromium-based framework), if you see that video acceleration is not present in Chrome, there's a good chance it's not present in the desktop app either for the same reasons. So we recommend getting things working in Chrome first, and if the desktop app is still having issues afterward then please file an issue and we'll look into it.

Reviewer: ```suggestion
You can check whether your platform supports hardware-accelerated video decoding by opening Google Chrome and entering `chrome://gpu` in the address bar. If you do not see `Video Decode: Hardware accelerated` or the `Video Acceleration` section is empty on this page, but you believe that it should be supported, then you may need to take additional platform-specific steps to enable it.

The desktop app uses Electron (based on Chromium), and should automatically use hardware acceleration when available. If you're seeing issues here, we recommend following the steps above to see if the same issue exists in Chrome. If so, there's a good chance it's affecting our application for the same reasons. We recommend getting things working in Chrome first, and if the desktop app is still having issues afterward please let us know and we'll look into it.
```

> ## Enabling video acceleration:

Reviewer: ```suggestion
## Enabling hardware acceleration
```

Author: "hardware acceleration" is technically correct however I've found better resources googling for this under "video acceleration" and also the `chrome://gpu` section that is referred to  for checking is "Video acceleration". So I'm going to leave this as is

> ## Enabling video acceleration:

Reviewer: ```suggestion
## Enabling video acceleration
```
